EnglishTranslation-master/SKILL.tsv

```diff
@@ -9690,195 +9690,195 @@ SKILL_20171128_009689	Shoot Lightning to enemy	적에게 뇌 탄을 발사합니
+SKILL_20171128_009773	Attack #{SkillFactor}#%{nl}Backward Movement Distance: #{CaptionRatio}#	공격력 #{SkillFactor}#%{nl}후방 이동거리 #{CaptionRatio}#	SKILL_20170420_008620	Attack #{SkillAtkAdd}#{nl}AoE Ratio #{SkillSR}#{nl}Reduced Enemy Movement Speed Chance #{CaptionRatio2}#%	공격력 #{SkillAtkAdd}#{nl}광역 공격 비율 #{SkillSR}#{nl}적 이동속도 감소 확률 #{CaptionRatio2}#%	
 SKILL_20171128_009774	Down Fall	다운 폴				
-SKILL_20171128_009775	{#DD5500}{ol}[Missile:Bow]{/}{/}{nl}Fires several arrows into the air and attacks arrows enemies a single designated enemy. If you and enemies are away from a certain distance, you will not be attacked.	{#DD5500}{ol}[미사일:활]{/}{/}{nl}여러발의 화살을 공중을 향해 발사하여 지정된 단일 적에게 화살 공격을 퍼붓습니다. 자신과 적이 일정 거리 이상 멀어지게되면 공격받지 않습니다.				
+SKILL_20171128_009775	{#DD5500}{ol}[Missile:Bow]{/}{/}{nl}Fire several arrows into the air and attacks arrows enemies a single designated enemy. If you and enemies are away from a certain distance, you will not be attacked.	{#DD5500}{ol}[미사일:활]{/}{/}{nl}여러발의 화살을 공중을 향해 발사하여 지정된 단일 적에게 화살 공격을 퍼붓습니다. 자신과 적이 일정 거리 이상 멀어지게되면 공격받지 않습니다.				
 SKILL_20171128_009776	Silver Bullet	실버불릿				
-SKILL_20171128_009777	All attacks using a pistol are given buffs which are changed to divine attributes.	권총을 사용하는 모든 공격이 신성 속성으로 변경되는 버프를 자신에게 부여합니다.				
-SKILL_20171128_009778	Duration #{CaptionTime}#seconds{nl}ble to use while riding	지속시간 #{CaptionTime}#초{nl}탑승 상태에서 사용 가능				
+SKILL_20171128_009777	All attacks using a pistol are augmented, changing them to Holy attribute.	권총을 사용하는 모든 공격이 신성 속성으로 변경되는 버프를 자신에게 부여합니다.				
+SKILL_20171128_009778	Duration: #{CaptionTime}# seconds{nl}Able to use while riding	지속시간 #{CaptionTime}#초{nl}탑승 상태에서 사용 가능				
 SKILL_20171128_009779	Tase	테이즈				
-SKILL_20171128_009780	{#DD5500}{ol}[Missile:Gun] - [Lightning]{/}{/}{nl}Whenever a target is damaged by putting the electric bullet into the enemy's body, the enemy gets additional electricity damage.	{#DD5500}{ol}[미사일:총] - [전기]{/}{/}{nl}전기가 통하는 탄을 적의 몸에 박아 넣어 대상이 대미지를 입을 때마다 전기 속성 추가 대미지를 입습니다.				
-SKILL_20171128_009781	Attack #{SkillFactor}#%{nl}Duration #{CaptionTime}# seconds{nl}Duration 10 times{nl}Able to use while riding	공격력 #{SkillFactor}#%{nl}지속시간 #{CaptionTime}#초{nl}지속 횟수 10 회{nl}탑승 상태에서 사용 가능				
+SKILL_20171128_009780	{#DD5500}{ol}[Missile:Gun] - [Lightning]{/}{/}{nl}Fire an electric bullet into the enemy's body, causing the target to take additional Lightning property damage when damaged.	{#DD5500}{ol}[미사일:총] - [전기]{/}{/}{nl}전기가 통하는 탄을 적의 몸에 박아 넣어 대상이 대미지를 입을 때마다 전기 속성 추가 대미지를 입습니다.				
```

Comment:
[Missile:Gun] -> [Missile: Gun] 

(I see that there are more lines having this, a simple mass replace could fix it.